manifests/globals.pp

```diff
@@ -12,11 +12,15 @@
 # @param enable_deb
 #   Enable debian content plugin
 #
+# @param enable_ansible_collection
+#   Enable ansible collection content plugin
```

Comment:
```suggestion
#   Enable ansible collection content plugin to sync and serve ansible content
```

Since we are also promoting ansible collections to manage Foreman/Katello config and operations, this wording will avoid potential user confusion

Reply:
Technically this goes for all these content plugin parts. Today they don't really enable any plugin but really configure Katello to enable/disable content handling. Perhaps we should have a follow up PR to clean up the phrasing? I haven't really looked too carefully at it since it's on the roadmap to remove (some technical challenges blocking it now) and now we've made sure that this isn't actually exposed to installer users.

Reply:
OK, thanks for the explanation. A follow up PR to clean up param phrasing sounds good to me. I'm sure there are other places in the installer module ecosystem where we would benefit as well, including some that are exposed to installer users.